### PR TITLE
docs: fix heritage links

### PR DESCRIPTION
**Pull request by jakovljevic-mladen**
Hi everyone,

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[x] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

I managed to find the similar issue that both Angular and RxJS documentation apps had. Basically, the docs app does not render what [EntityCollectionServiceBase](https://ngrx.io/api/data/EntityCollectionServiceBase) implements. While the docs app says this:

`class EntityCollectionServiceBase<...> implements EntityCollectionService`,

[it actually implements this](https://github.com/ngrx/platform/blob/d3aa7d0bde6c0e4de4b3e91a99fa6708b8c341bb/modules/data/src/entity-services/entity-collection-service-base.ts#L31):

`class EntityCollectionServiceBase<...> implements EntityCollectionService<T>`.

Basically, the docs do not render generic type `<T>`. Here's how this looks like in the DOM:

image

## What is the new behavior?
This PR fixes the docs rendering for similar cases. This is how this looks like with this fix:

image

p.s. I added what Angular docs already have: the check if `clause.doc.path` exists. If it exists, make sure to add `<a>` tag, if it doesn't, just render `clause.text`.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

## Other information
For more info, please check these related issues/PRs:
Related angular/angular#43956
Related ReactiveX/rxjs#6655
Related ReactiveX/rxjs#6427

### CODE REVIEW

**Review by markostanimirovic — APPROVED** [review]
Thanks Mladen! 👏

**Review by timdeschryver — APPROVED** [review]
Thanks @jakovljevic-mladen - you also get a 👍 for the detailed description with screenshots.
